### PR TITLE
EX-577: RTCM GLO EPH health bits

### PR DESCRIPTION
**Pull request by nsirola**
- Fix RTCM GLO ephemeris decoder to check all threee health bits.
- Check the GLO "Additional Data" flag and read additional data only if set.
- Remove the unused field `valid` from `rtcm_msg_eph` (is not read anywhere and only set in the BDS ephemeris decoder)
- Fix the function comments return values to match the code

# Testing

Passes `gnss-converters` unit tests.

Btw in the RTCM logs run through the other unit tests there are some cases of ephemeris for GLO PRN 26 (flight test?) that has only the `Bn` health flag on and was thus incorrectly marked healthy before.

### CODE REVIEW

File: c/src/eph_decode.c [hunk 1]
```diff
@@ -98,7 +97,6 @@ rtcm3_rc rtcm3_decode_gps_eph(const uint8_t buff[], rtcm_msg_eph *msg_eph) {
  * \param RTCM message struct
  * \return  - RC_OK : Success
  *          - RC_MESSAGE_TYPE_MISMATCH : Message type mismatch
- *          - RC_INVALID_MESSAGE : Cell mask too large or invalid TOW
  */
 rtcm3_rc rtcm3_decode_glo_eph(const uint8_t buff[], rtcm_msg_eph *msg_eph) {
```
Comment (mbavaro) [inline c/src/eph_decode.c]: can we add 
```
  msg_eph->constellation = RTCM_CONSTELLATION_GLO;
```
here too?

Comment (mbavaro) [reply c/src/eph_decode.c]: and in general also something like
```suggestion
rtcm3_rc rtcm3_decode_glo_eph(const uint8_t buff[], rtcm_msg_eph *msg_eph) {
  assert(msg_eph);
  memset(msg_eph, 0, sizeof(rtcm_msg_eph));
```

**Comment by nsirola** [comment]
Ok, I'll merge this and add a follow-up with asserts and proper initialization of each array as those changes requires some more testing.

**Comment by nsirola** [comment]
Added a follow-up issue https://swift-nav.atlassian.net/browse/EX-578

**Pull request merged**